### PR TITLE
[4.0] Completely remove tooltips from forms [a11y]

### PR DESCRIPTION
**Pull request by brianteeman**
This PR completely removes tooltips from form field labels. No more hover actions etc

The previously displayed form field descriptions now have an id so that the field input can state that it is aria-describedby the description if it exists

Easiest place to test if with com_banners
`administrator/index.php?option=com_banners&view=banner&layout=edit`

On the right you can see the description for the "Pinned" switch
Before the PR the description is also available when you hover over the label
After the PR there is no more hover action AND if you inspect the code you will see the new aria-describedby code

### CODE REVIEW

File: layouts/joomla/form/renderfield.php [hunk 1]
```diff
@@ -26,18 +29,21 @@
...
 $rel   = empty($options['rel']) ? '' : ' ' . $options['rel'];
+$id    = $name . '-desc';
+
 ?>
 <div class="control-group<?php echo $class; ?>"<?php echo $rel; ?>>
 	<?php if (empty($options['hiddenLabel'])) : ?>
 		<div class="control-label"><?php echo $label; ?></div>
 	<?php endif; ?>
-	<div class="controls"><?php echo $input; ?></div>
+	<div class="controls"<?php if (!empty($description)) : ?> aria-describedby="<?php echo $id; ?><?php endif; ?>">
```
Comment (Quy) [inline layouts/joomla/form/renderfield.php]: Move the last `"` inside the `if` statement.

File: layouts/joomla/form/renderfield.php [hunk 2]
```diff
@@ -26,18 +29,21 @@
...
 	<?php endif; ?>
-	<div class="controls"><?php echo $input; ?></div>
+	<div class="controls"<?php if (!empty($description)) : ?> aria-describedby="<?php echo $id; ?>"<?php endif; ?>>
+		<?php echo $input; ?>
+	</div>
 	<?php if (!empty($description)) : ?>
-		<div>
-			<small class="form-text text-muted"><?php echo $description; ?></small>
+		<div id="<?php echo $id; ?>">
+			<small class="form-text text-muted"><?php echo $description; ?></small>
```
Comment (Quy) [inline layouts/joomla/form/renderfield.php]: Remove trailing space

**Comment by Quy** [comment]
It would be nice if it has the same syntax as label id such as `jform_sticky-desc`.

```
			<div class="control-label"><label id="jform_sticky-lbl" >
	Pinned</label>
</div>
		<div class="controls" aria-describedby="jform[sticky]-desc">
```

**Comment by brianteeman** [comment]
It has to be the same as the id of the input

**Comment by brianteeman** [comment]
Closed - obviously no one else is interested in improving the accessibility in joomla 4

**Comment by brianteeman** [comment]
Trying again - maybe someone will look

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**